### PR TITLE
fix(service-portal): service portal - petitions

### PR DESCRIPTION
**Pull request by albinagu**
Screenshot 2023-02-27 at 14 21 51
Screenshot 2023-02-27 at 14 22 13
Screenshot 2023-02-27 at 14 22 44

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/petitions/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- gitbook-ignore -->
+
+# Service Portal Petitions
+
+This module displays what petitions are assigned to you, and allows you to un-register your petitions or register new petitions
+
+## Running unit tests
+
+Run `nx test service-portal-petitions` to execute the unit tests via [Jest](https://jestjs.io).
```
Comment (alexdiljar) [inline libs/service-portal/petitions/README.md:9]: Maybe add to readme that you need to run endorsement service to test locally?

File: libs/service-portal/petitions/src/screens/PetitionsAdmin/index.tsx [hunk 2]
```diff
@@ -42,21 +42,13 @@ const PetitionsAdmin = () => {
...
-        </BulletList>
-      </Box>
+      <IntroHeader title={m.title} intro={m.intro} />
 
       <Box marginTop={5} marginBottom={7}>
         {openLists && openLists.length > 0 && (
           <>
             <Text as="p" variant="h4" marginBottom={2}>
-              {formatMessage(m.petition.petitionListsOngoing)}
+              {'Lorem ipsum'}
```
Comment (thordurhhh) [inline libs/service-portal/petitions/src/screens/PetitionsAdmin/index.tsx:51]: Lorem ipsum 👀

File: libs/service-portal/petitions/src/screens/PetitionsAdmin/index.tsx [hunk 3]
```diff
@@ -97,7 +89,7 @@ const PetitionsAdmin = () => {
         {closedLists && closedLists.length > 0 && (
           <>
             <Text as="p" variant="h4" marginBottom={3} marginTop={7}>
-              {formatMessage(m.petition.petitionListsClosed)}
+              {'Lorem ipsum'}
```
Comment (thordurhhh) [inline libs/service-portal/petitions/src/screens/PetitionsAdmin/index.tsx:92]: 👁️

File: libs/service-portal/petitions/src/screens/PetitionsTable/ExportPetition/downloadCSV.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  name: string,
+  header: string[],
+  data: Array<Array<string | number>>,
+  type: 'csv',
+) => {
+  const getFile = (name: string, output: string | undefined) => {
+    const uri =
+      type === 'csv'
+        ? 'data:text/plain;charset=utf-8,\uFEFF'
+        : 'data:application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;base64,'
```
Comment (thordurhhh) [inline libs/service-portal/petitions/src/screens/PetitionsTable/ExportPetition/downloadCSV.ts:13]: If you don't need the Excel format, the csvStringify function on line 25 could be simplified to:

```
    CSVStringify(data, (err, output) => {
      const encodedUri = encodeURI(`data:text/csv;charset=utf-8,${output}`)
      const link = document.createElement('a')
      link.setAttribute('href', encodedUri)
      link.setAttribute('download', name)
      document.body.appendChild(link)

      link.click()
    })
```

**Review by thordurhhh — CHANGES_REQUESTED** [review]
Just a few minor things in the comments.

File: libs/service-portal/petitions/src/screens/ViewPetitionAdmin/index.tsx [hunk 5]
```diff
@@ -193,66 +188,48 @@ const ViewPetitionAdmin = () => {
             {!petition.adminLock ? (
               <DialogPrompt
                 baseId="demo_dialog"
-                title={formatMessage(m.viewPetition.dialogPromptLockListTitle)}
-                ariaLabel={formatMessage(
-                  m.viewPetition.dialogPromptLockListTitle,
-                )}
+                title={'Lorem ipsum'}
+                ariaLabel={'Lorem ipsum'}
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/petitions/src/screens/ViewPetitionAdmin/index.tsx:192]: this functionality will be moved to the admin system, so i left these texts out while refactoring messages :3

[the remark this answers is not in the document]
Comment (thordurhhh) [reply libs/service-portal/petitions/src/screens/ViewPetitionAdmin/index.tsx:192]: ahh alright

File: libs/service-portal/petitions/README.md (hunk 1 above)
Comment (alexdiljar) [reply libs/service-portal/petitions/README.md:9]: Lets do it when er overhaul the whole documentation for the endorsement system :)

**Review by alexdiljar — COMMENTED** [review]
A couple of hard coded strings in there otherwise good

**Review by alexdiljar — APPROVED** [review]

**Review by saevarma — APPROVED** [review]
Core files LGTM

**Pull request merged**